### PR TITLE
George

### PR DESCRIPTION
**Pull request by iamNanaKwame**

### CODE REVIEW

File: models/product.model.js [hunk 1]
```diff
@@ -1 +1,34 @@
...
+		},
+		createdBy: {
+			type: String,
+			required: true,
+		},
+		createdAt: {
+			type: Date,
+			required: true,
+		},
+		path: {
```
Comment (owuraku) [inline models/product.model.js:27]: what does path signify ? I thought we use the ID of the uploaded file for images. @iamNanaKwame

File: models/product.model.js [hunk 2]
```diff
@@ -1 +1,34 @@
...
+		},
+		quantity: {
+			type: Number,
+			default: 0,
+		},
+		createdBy: {
+			type: String,
+			required: true,
+		},
+		createdAt: {
```
Comment (owuraku) [inline models/product.model.js:23]: default for date ?

**Review by owuraku — CHANGES_REQUESTED** [review]
We decided to use feature branches, that way we understand the work done for the PR (pull request). So the branch shouldn't be George but rather the task id from Jira. 
Check the details of the [task]( https://nestle-node.atlassian.net/browse/NES-1) using this link :  https://nestle-node.atlassian.net/browse/NES-1 

it has subtasks as well so ideally, the PR should be more than this. it should be a complete CRUD for the Product model. 

In the end, the PR should be like this **NES-1-Add Product CRUD** in the description you can add a link to the issue on JIRA to help anyone who wants to test the code

@iamNanaKwame

**Comment by iamNanaKwame** [comment]
Recalled ... I pushed it to ask the team if I had left something out.
Dropped them a message on whatsapp for feedback but then, will use the
model agreed upon. Getting my hands into it.

Thanks

**Pull request closed without merging**